### DISCUSSION TITLE
Empty PDF Certificate on PU

### DISCUSSION THREAD

**Discussion by nicolaslang1** [Digitaler-Impfnachweis/certification-apis#126] [Q&A] [answered]
Hello,

we are on PU and by calling the api we receive an empty PDF. 

The PDF is attached.

[cert.pdf](https://github.com/Digitaler-Impfnachweis/certification-apis/files/6686072/cert.pdf)

**Comment by thinkberg** [comment, marked as answer]
What type of certificate was expected?

**Reply by nicolaslang1** [reply]
I am sorry. I passed the response body to a stream reader and the streamreader broke the file. So i tried to directly write the response body to a pdf file and now it works.
